### PR TITLE
Merge translation files from assisted-ui-lib into one file

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://github.com/openshift-assisted/assisted-ui-lib/pull/1399/

Related to https://issues.redhat.com/browse/MGMT-8156

Signed-off-by: Montse Ortega <mortegag@redhat.com>

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@ammont82 Is this a work in progress? I have some questions and potentially some changes to request once it is ready for review.

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
Thanks Montse! This looks great.

I think the files under `locales_merged` should be added to `.gitignore` and not committed.

2 other suggestions made inline.

File: frontend/webpack.plugin.ts [hunk 1]
```diff
@@ -81,13 +82,33 @@ module.exports = function (env: any, argv: { hot?: boolean; mode: string | undef
...
+              new MergeJsonWebpackPlugin({
+                files: [
+                  "../../public/locales/en/translation.json",
+                  "../../node_modules/openshift-assisted-ui-lib/dist/locales/en/translation.json"
+                ],
+                output: {
+                  "fileName": "../../../public/locales_merged/en/translation.json"
+                },
+                space: 4
+              }),
```
[the remark this answers is not in the document]
Comment (ammont82) [reply frontend/webpack.plugin.ts:105]: Yes, I can review and try to handle each locale

File: package.json [hunk 2]
```diff
@@ -54,5 +54,8 @@
         "tabWidth": 4,
         "semi": false,
         "singleQuote": true
+    },
+    "dependencies": {
+        "merge-jsons-webpack-plugin": "^2.0.1"
```
[the remark this answers is not in the document]
Comment (ammont82) [reply package.json:59]: Ok, I can change

File: frontend/webpack.plugin.ts [hunk 3]
```diff
@@ -81,7 +82,19 @@ module.exports = function (env: any, argv: { hot?: boolean; mode: string | undef
...
-            }),
+            }),       
+            ...locales.map((locale) => {
+                return new MergeJsonWebpackPlugin({
+                    files: [
+                        `../../public/locales/${locale}/translation.json`,
+                        `../../node_modules/openshift-assisted-ui-lib/dist/locales/${locale}/translation.json`
+                      ],
+                    output: {
+                    "fileName": `../../../public/locales/${locale}/translation.json`
```
Comment (rawagner) [inline frontend/webpack.plugin.ts:93]:
```suggestion
                        "fileName": `../../../public/locales/${locale}/translation.json`
```

**Pull request merged**